pkg/kubelet/volume/nfs/nfs_mount.go

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright 2014 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package nfs
+
+import (
+	"os/exec"
+	"syscall"
+
+	"github.com/golang/glog"
+)
+
+type nfsMountInterface interface {
+	// Mount wraps exec.Command for an NFS Mount
```

Comment:
Don't document the interface in terms of the implementation.  Document what Mount() should do here, and describe how it works (and why, please) on the implementation of this function.